src/codeu/chat/client/commandline/Chat.java

```diff
@@ -491,6 +498,111 @@ private ConversationHeader findConversation(String title) {
     return null;
   }
 
+  private Panel createPlayPanel(final UserContext user) {
+
+    final Panel panel = new Panel();
+
+    panel.register("help", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        System.out.println("  titles");
+        System.out.println("    Display titles of all available plays.");
+        System.out.println("  all");
+        System.out.println("    Display all ongoing plays and their statuses.");
+        System.out.println("  new <title>");
+        System.out.println("    Create and join a new play enactment for the given title.");
+        System.out.println("  join <title>");
+        System.out.println("    Join an ongoing play enactment for the given title.");
+        System.out.println("  back");
+        System.out.println("    Go back to the user panel.");
+      }
+    });
+
+    panel.register("titles", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        for (final String title : context.allPlayTitles()) {
+          System.out.println(title);
+        }
+      }
+    });
+
+    panel.register("all", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        for (final PlayInfo p : context.allPlays()) {
+          System.out.format("%s - %s\n", p.getTitle(), p.getStatus());
+        }
+      }
+    });
+
+    panel.register("new", new Panel.Command() {
+      @Override
+      public void invoke(List<String> args) {
+        StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < args.size(); i++) {
+          builder.append(args.get(i));
+          builder.append(" ");
+        }
+        builder.deleteCharAt(builder.length() - 1);
```

Comment:
Since you are re-using this StringBuilder logic multiple times, you could write this into a tiny private static helper function at the end of this class.
```
private static String getTitleFromArgs(List<String> args) {
  StringBuilder builder = new StringBuilder();
  for (int i = 0; i < args.size(); i++) {
    builder.append(args.get(i)).append(" ");
  }
  builder.deleteCharAt(builder.length() - 1);
  return builder.toString();
}
```